app/src/main/java/swati4star/createpdf/activity/MainActivity.java

```diff
@@ -439,4 +451,12 @@ private boolean getRuntimePermissions() {
         }
         return true;
     }
+
+    public void convertZipToPdf(ArrayList<Uri> imageUris) {
```

Comment:
add javadoc style comments to this function

Reply:
okk